website/sidebars.json

```diff
@@ -66,6 +66,10 @@
     ],
     "Installation": [
       "boilerplate/boilerplate-instalation"
+    ],
+    "Features": [
+      "boilerplate/boilerplate-custom-fonts",
+      "boilerplate/boilerplate-icon-fonts"
```

Comment:
I think these should go in a separate "Examples" category as we talked about on standup (as I've set it up in #13, I'd rather keep Theme & Plugin pages like a github readme - 1 page). As far as I'm concerned you can add `Examples` to the main header menu anywhere for now if you want.